tests/ts_simple_query_buffers_SUITE.erl

```diff
@@ -393,3 +392,10 @@ col_no({[A|_], _, _}) ->
 
 fmt(F, A) ->
     lists:flatten(io_lib:format(F, A)).
+
+get_write_perm_cmd(take_away, QBufDir) ->
+    ct:pal("take away perms on ~s\n", [QBufDir]),
+    fmt("sh -c \"mv '~s' '~s.boo' && touch '~s'\"", [QBufDir, QBufDir, QBufDir]);
```

Comment:
I have no strong preference here. It's just a very expressive one-liner bash job compared with double (gasp) the number of lines when done using native Erlang means. It would matter if we run the tests on a Windows box (which we are not going to do I suppose :)